plugins/modules/nxos_bgp_af.py

```diff
@@ -213,6 +213,14 @@
     - present
     - absent
     type: str
+  retain_route_target:
```

Comment:
Please add a `version_added: 1.1.0` key to all the new attributes.

Reply:
Added version